app/src/main/res/layout/story_chapter_view.xml

```diff
@@ -91,8 +92,7 @@
             android:layout_marginBottom="4dp"
             android:ellipsize="end"
             android:fontFamily="sans-serif"
-            android:maxLines="4"
-            android:text="@{htmlContent}"
+            android:text="@{viewModel.chapterSummary.chapterPlayState != ChapterPlayState.NOT_PLAYABLE_MISSING_PREREQUISITES ? htmlContent : String.format(@string/chapter_prerequisite, viewModel.index)}"
```

Comment:
We also need to show chapter name and it's in Roboto Medium, #555555

<img width="328" alt="Screenshot 2020-12-07 at 08 55 21" src="https://user-images.githubusercontent.com/25057618/101306057-f64e9a00-3869-11eb-9dcc-25eb37c47c7a.png">

<img width="296" alt="Screenshot 2020-12-07 at 08 57 41" src="https://user-images.githubusercontent.com/25057618/101306214-4af21500-386a-11eb-80ab-22c22f88bf8a.png">

Reply:
https://github.com/oppia/oppia-android/blob/f616e8570ee6c791250a0675e0b728204b96b1d1/app/src/main/java/org/oppia/android/app/story/StoryViewModel.kt#L102

1. We can check how we can get the name of the previous chapter because of which any chapter is locked and access it and make it a part of the ViewModel (`StoryChapterSummaryViewModel`) item which we are using to show a locked item. It's like introducing a new parameter `previousChapter`
2. If there is any other way you can think which best fit here on how to access the previous chapter name, we can use it too.

Reply:
Done. Added a new parameter `previousChapterTitle`.
Regarding Roboto Medium, #555555 - is it for the whole prerequisite? <b>Chapter title</b> is being ignored, not sure why?

Reply:
Html tags in strings.xml don't work with `android:text`. What's the preferred alternative? @anandwana001

Reply:
It looks like this is some edge case where styling the passed string need some extra things. 
Please have a look here - https://developer.android.com/guide/topics/resources/string-resource

Reply:
I see that I need to use `Html.fromHtml`, but I don't think I can use it in xml file, can I?

Reply:
I don't think so, as per our discussion in the meeting, we can do it kotlin side.

Reply:
Resolving this as we can skip this part from this PR 
reference - https://github.com/oppia/oppia-android/pull/2232#discussion_r541472919